src/Enterspeed.Source.SitecoreCms.V9/Events/PublishEndEventHandler.cs

```diff
@@ -22,7 +22,7 @@ public void PublishEnd(object sender, EventArgs args)
 
             object[] parameters = { batchSizeParsed ? 2000 : batchSize };
 
-            var jobOptions = new Sitecore.Jobs.DefaultJobOptions("handleQueuedJobs", "Enterspeed", "CM", _enterspeedJobsHandlingService, "HandlePendingJobs", parameters)
+            var jobOptions = new Sitecore.Jobs.DefaultJobOptions("handleQueuedJobs", "Enterspeed", Sitecore.Context.GetSiteName(), _enterspeedJobsHandlingService, "HandlePendingJobs", parameters)
```

Comment:
Why are we parsing the ´Sitecore.Context.GetSiteName()´ compared to previous where it was a static value of `CM`? Just for me to understand :-)